bitnami/kubernetes-event-exporter/templates/deployment.yaml

```diff
@@ -72,19 +102,33 @@ spec:
                 name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsSecret "context" $) }}
             {{- end }}
           {{- end }}
-          resources: {{- toYaml .Values.resources | nindent 12 }}
-          {{- if .Values.containerSecurityContext.enabled }}
-          # yamllint disable rule:indentation
-          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
-          # yamllint enable rule:indentation
+          {{- if not .Values.diagnosticMode.enabled }}
+          {{- if .Values.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.startupProbe.enabled }}
+          startupProbe: {{- omit .Values.startupProbe "enabled" | toYaml | nindent 12 }}
+          {{- else if .Values.customStartupProbe }}
+          startupProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customStartupProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.livenessProbe.enabled }}
+          livenessProbe: {{- omit .Values.livenessProbe "enabled" | toYaml | nindent 12 }}
```

Comment:
Which is the check for the liveness probe?

Reply:
All probes are disabled by default.

I was thinking about not adding probes at all since the chart doesn't even expose a containerPort. But then I saw we already do this in the `bitnami/kubewatch` chart, which I'm also standardising:
https://github.com/bitnami/charts/blob/dd28dccabfa9d9e917a9e12a6b8d887837f0cd87/bitnami/kubewatch/templates/deployment.yaml#L84-L93

Since it is interpreted as a template, users could define their probe command under `livenessProbe`, but imho it doesn't make sense to have both `livenessProbe` and `customLivenessProbe`.

What do you think would fit in both cases?

Reply:
I would keep only `customXXX` removing `livenessProbe`, `readinessProbe` and `startupProbe` parameters in this case.

Reply:
I will then update `kubewatch` PR too.